wasm/src/main/java/com/dylibso/chicory/wasm/types/FunctionType.java

```diff
@@ -4,21 +4,21 @@
 import java.util.Objects;
 
 public final class FunctionType {
-    private final List<ValueType> params;
-    private final List<ValueType> returns;
+    private final List<NewValueType> params;
+    private final List<NewValueType> returns;
     private final int hashCode;
 
-    private FunctionType(List<ValueType> params, List<ValueType> returns) {
+    private FunctionType(List<NewValueType> params, List<NewValueType> returns) {
         this.params = params;
         this.returns = returns;
         hashCode = Objects.hash(params, returns);
     }
 
-    public List<ValueType> params() {
+    public List<NewValueType> params() {
         return params;
     }
 
-    public List<ValueType> returns() {
+    public List<NewValueType> returns() {
         return returns;
     }
```

Comment:
We shouldn't be changing existing public methods.  These should continue to return List<ValueType>, right?

Reply:
`FunctionType` is not really used in user code, up to this change is basically an internal data structure.

Reply:
as an aside, are you just checking the projects here https://github.com/hypercubestart/chicory?tab=readme-ov-file#who-uses-chicory to understand which parts of the API are used publically?

Reply:
good idea, might be worth checking some dependants too. I'll check chicory-sdk